tedium.js

```diff
@@ -245,14 +342,23 @@ Promise.resolve().then(() => {
         })
     );
   }
-  return promiseAllWithProgress(promises, 'Cleaning...');
+  return promiseAllWithProgress(promises, 'Applying transforms...');
 }).then(() => {
-  if (elementsPushed === 0) {
+  console.log();
+  if (elementsPushed === 0 && pushesDenied === 0) {
     console.log('No changes needed!');
-  } else {
+  } else if (pushesDenied === 0) {
     console.log(`Successfully pushed to ${elementsPushed} repos.`);
+  } else if (opts.max_changes === 0) {
+    console.log(`${pushesDenied} changes ready to push. ` +
+                `Call with --max_changes=N to push them up!`);
+  } else {
+    console.log(`Successfully pushed to ${elementsPushed} repos. ` +
+                `${pushesDenied} remain.`);
   }
 }, function(err) {
-  console.error(`\n\n${err}`);
+  console.error('\n\n');
+  console.error(err);
```

Comment:
Prooobably, but passing multiple arguments to console.log and friends sometimes does weird stuff.

Compare `console.log('\n', {});` with `console.log({}, '\n');`

They produce:

```

{}
```

And

```
{} '\n'
```

Respectively. So I like separate console calls.